03_letters/CameronSchafer/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+// string to rune with counts
```

Comment:
General go convention when adding comments to functions is that they start with the name of the function and then an explanation. This was the comment is picked up in the godoc.

```go
// letters returns a count of each letter in the given string
func letters(s string) []string {
    ...
}
```

Can you update this?

Reply:
Done